### DISCUSSION TITLE
Config files for latest JyersUI bugfix build

### DISCUSSION THREAD

**Discussion by PrintedVisionz3D** [Jyers/Marlin#1582] [Q&A]
I am trying to compile a build for my ender 5 pro but it tells me that my configuration_adv.h file is for an old version of Marlin, is there a newer version of the config file for the latest bugfix build?

**Comment by Cal-LF** [comment]
Configuration Files/E3V2 Templates to choose which config to start with.

**Reply by kted** [reply]
The default config has 
`CONFIGURATION_H_VERSION 02000903`, while the ones in Templates have 
`CONFIGURATION_H_VERSION 02000902`. Hence the error. 
Just by changing the vesion in both config files to 02000903, compiles OK, but I haven't tested for operation...

**Comment by Cal-LF** [comment]
The configs are updated to the new version. Your compile should work.

**Reply by PrintedVisionz3D** [reply]
I think I see the problem is, I'm trying to compile for my Ender 5 pro but I've been told that this firmware is only for Ender 3
